Sources/MPDB.swift

```diff
@@ -17,7 +17,7 @@ class MPDB {
     let apiToken: String
     
     init(token: String) {
-        apiToken = token.filter { !$0.isWhitespace }
```

Comment:
can we add a couple of tests for this?

Reply:
modified `testMPDB()` to use a token with white space and a bunch of special chars